frontend/site-registry/src/app/features/details/SiteDetails.tsx

```diff
@@ -587,9 +594,7 @@ const SiteDetails = () => {
           items={navItems}
           components={navComponents}
           dropdownItems={dropDownNavItems}
-          isDisable={false
-           
-          }
+          isDisable={false}
```

Comment:
isDisable = {
            UserType.External === userType && snapshot.snapshot.data === null
          }

Please add this condition as navigation pills will not be available everytime. it will be enable and disable based of snapshot is purchased or not.

Reply:
Sure, added